test/CMakeLists.txt

```diff
@@ -161,7 +161,11 @@ target_compile_definitions(ArborX_Clustering.exe PRIVATE BOOST_TEST_DYN_LINK)
 target_include_directories(ArborX_Clustering.exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_SOURCE_DIR}/examples/dbscan)
 add_test(NAME ArborX_Clustering_Test COMMAND ./ArborX_Clustering.exe)
 
-add_executable(ArborX_DetailsTreeNodeLabeling.exe tstDetailsTreeNodeLabeling.cpp utf_main.cpp)
+add_executable(ArborX_DetailsTreeNodeLabeling.exe
+  tstDetailsTreeNodeLabeling.cpp
+  tstDetailsMutualReachabilityDistance.cpp
```

Comment:
Mutual reachability has nothing to do with labels, though. Should probably be its own test.

Reply:
We should update the executable name indeed but I would still have then compiled into the same exe.  Do you have an issue with that?

Reply:
I renamed the executable.